app/pages/sections/order-items/catalogue-solutions/routes.js

```diff
@@ -53,15 +53,13 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
   router.get('/:orderItemId', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
     const { orderId, orderItemId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
-
```

Comment:
We don't need to delete this blank line.

Reply:
Added blank line.